packages/angular/src/generators/ngrx-root-store/lib/validate-options.ts

```diff
@@ -0,0 +1,57 @@
+import type { Tree } from '@nx/devkit';
+import { getProjects, readProjectConfiguration } from '@nx/devkit';
+import { Schema } from '../schema';
+import {
+  getInstalledAngularVersionInfo,
+  getInstalledPackageVersionInfo,
+} from '../../utils/version-utils';
+import { getPkgVersionForAngularMajorVersion } from '../../../utils/version-utils';
+import { coerce, lt, major } from 'semver';
+import { isNgStandaloneApp } from '../../../utils/nx-devkit/ast-utils';
+
+export function validateOptions(tree: Tree, options: Schema): void {
+  if (!options.project) {
+    throw new Error('Please provide a value --project');
+  }
```

Comment:
nit: we don't need this check. The schema and the type definition require the `project` option.